src/render-help.js

```diff
@@ -55,6 +55,12 @@ module.exports = async () => {
   output.writeText(colors.gray('Global options'));
   output.writeText(formatLine('--verbose', 'Enable verbose logs'));
   output.writeText(formatLine('--stage', 'Stage of the service'));
+  output.writeText(
+    formatLine(
+      '--max-concurrency',
+      'Specify maximum number of concurrently running service methods'
```

Comment:
```suggestion
      'Specify the maximum number of concurrently running service commands'
```

(methods doesn't mean anything in the users perspective)